commands/create_test.go

```diff
@@ -90,15 +92,23 @@ func Test_CreateSupportBundle(t *testing.T) {
 			conf := &supportBundleCommandConfiguration{
 				caseNumber: "1234",
 			}
-			now := func() string { return "now" }
-			id, err := createSupportBundle(&test.given, conf, now)
+			clock := func() time.Time {
```

Comment:
you could do `time.Unix(0, 0)` instead.

Reply:
With `time.Parse`, we can make a connection between this value and what is in the SB description.